graphql.py

```diff
@@ -99,6 +99,7 @@ def fetch_user_list(username: str):
                         id
                         progress
                         status
+                        score
```

Comment:
```suggestion
                        score(format:POINT_100)
```
Otherwise, the score will be returned according to the user's setting.